plugins/uefi/fu-plugin-uefi.c

```diff
@@ -29,7 +29,8 @@
 #include "fu-plugin.h"
 #include "fu-plugin-vfuncs.h"
 
-#ifndef UX_CAPSULE_GUID
+/* drop when upgrading minimum required version of efivar to 33 */
+#ifndef HAVE_EFIVAR_UX_CAPSULE
```

Comment:
Can't we just change the #define to `efi_guid_ux_capsule` and avoid a lot of #ifdefs?

Reply:
Nope, we can't have nice things because efivar 32 in fedora, debian, ubuntu are all patched to recognize that.

Reply:
Then can't we do the opposite e.g. #define HAVE_EFIVAR_UX_CAPSULE efi_guid_ux_capsule?